index.js

```diff
@@ -0,0 +1,174 @@
+const express = require('express');
+const crypto = require('crypto');
+var execFile = require('child_process').execFile;
+var exec = require('child_process').exec;
+const app = express();
+var formidable = require('formidable');
+var path = require('path');
+var fs = require('fs');
+var archiver = require('archiver')
+
+app.use(express.json())
+app.get('/v1/obabel', (req, res) => {
+    responseIsSent = false;
+    jobId = req.query.storage_hash;
+    uploadsPath = path.join(__dirname,'uploads');
+    fullPath = path.join(__dirname,"uploads", jobId);
+    
+    obabelOutputFilePath = path.join(fullPath,'obabel-output.txt')
+    if (!fs.existsSync(fullPath) && !responseIsSent) {
```

Comment:
So responseIsSent will always be false here. It looks like you're trying to solve a concurrency issue with this logic?

Reply:
There was an issue where the program would attempt to send a response to the same request twice (causing an error). I originally added superfluous checks for the sake of futureproofing/consistency of logic, but I have since removed some of the more obviously redundant ones (like this one).